utils/create_dummy_models.py

```diff
@@ -67,6 +66,9 @@
 if not is_tf_available():
     raise ValueError("Please install TensorFlow.")
 
+from get_test_info import get_model_to_tester_mapping, get_tester_classes_for_model  # noqa E402
```

Comment:
Why can't this be with the other imports above?
Note: can't comment on the sys.path(".") above but the current folder is always in the path, so it's not necessary.

Reply:
@sgugger In a few files under `utilis`, if there are statements like `importlib.import_module("tests/models/...")`, it requires adding
```python
# This is required to make the module import works (when the python process is running from the root of the repo)
sys.path.append(".")
```
Otherwise, I get `ModuleNotFoundError: No module named 'tests'`.

The file `utils/create_dummy_models.py` had this `importlib.import_module(...)` before this PR, so it required `sys.path.append(".")`. I can remove it from this PR.

(I think I should try to figure out why we need it in the case mentioned above - but if you happen to know, please teach me a lesson 🙏 )

Reply:
> Why can't this be with the other imports above?

Probably yes, I will try! I am not good at import/export

Reply:
Ah yes, that's why it's there then. Could you make sure the comment is added so we now why this sys.path.apoend is there?

Reply:
It's no longer needed for this file `create_dummy_models.py` (as I re-used the method in another file).
In another file (`get_test_info.py`), I have a comment 
```python
# This is required to make the module import works (when the python process is running from the root of the repo)
```